notebooks/deepdive/custom/ucaip_customjob.ipynb

```diff
@@ -0,0 +1,1857 @@
+{
```

Comment:
<p>This sentence is outdated:</p><p> </p><p>"You can then&nbsp;create an AI Platform model version based on this output in order to serve&nbsp;online predictions."</p><p> </p><p>If this is in your notebook template for AI Platform (Unified) notebooks, please update that to replace "model version" with "endpoint", in the template and in all affected notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>